react/ProductSummaryName.tsx

```diff
@@ -64,6 +68,8 @@ function ProductSummaryName({
         brandNameClass={brandNameClasses}
         skuNameClass={skuNameClasses}
         loaderClass={loaderClasses}
+        showSponsoredBadge={isSponsored}
```

Comment:
Lint will fail here because the current version of `store-components` does not support the props `showSponsoredBadge` and `sponsoredBadgeLabel`. Once https://github.com/vtex-apps/store-components/pull/1098 is published, I'll change the package.json and this should be fixed.